net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+			<fieldname>sleep</fieldname>
+			<description>Amount of time in seconds to sleep between processing spool files.</description>
+			<default_value>5</default_value>
+			<type>input</type>
+			<size>3</size>
+			<required>true</required>
+		</field>
+		<field>
+			<fielddescr>Use Trap Time</fielddescr>
+			<fieldname>use_trap_time</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have SNMPTT use the time that the trap was processed by SNMPTTHANDLER<br/>
+					Set to 0 to have SNMPTT use the time the trap was processed.  Note:  Using 0 can<br/>
+					result in the time being off by the number of seconds used for 'sleep'<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Keep Unlogged Traps</fielddescr>
+			<fieldname>keep_unlogged_traps</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to have SNMPTT erase the spooled trap file after it attempts to process<br/>
+					the trap even if it did not successfully log the trap to any of the log systems.<br/>
+					Set to 1 to have SNMPTT erase the spooled trap file only after it successfully<br/>
+					logs to at least ONE log system.<br/>
+					Set to 2 to have SNMPTT erase the spooled trap file only after it successfully<br/>
+					logs to ALL of the enabled log systems.  Warning:  If multiple log systems are<br/>
+					enabled and only one fails, the other log system will continuously be logged to<br/>
+					until ALL of the log systems function.<br/>
+					The recommended setting is 1 with only one log system enabled.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Duplicate Trap Window</fielddescr>
+			<fieldname>duplicate_trap_window</fieldname>
+			<description>How often in seconds duplicate traps will be processed.</description>
+			<default_value>0</default_value>
+			<type>input</type>
+			<size>3</size>
+			<required>true</required>
+		</field>
+		<field>
+			<name>Logging</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Stdout Enable</fielddescr>
+			<fieldname>stdout_enable</fieldname>
+			<description>Set to 1 to enable messages to be sent to standard output, or 0 to disable.</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Log Enable</fielddescr>
+			<fieldname>log_enable</fieldname>
+			<description>Set to 1 to enable text logging of TRAPS.</description>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.